### PR TITLE
Make Browserify bundle work in ES3 environments

### LINKED ISSUE #90: Internet Explorer 8 and Browserify
Simply using `require('intl-messageformat');` in a bundle that is browserified results in JavaScript errors in IE8. The errors occur as a result of the keyword `default` being used as a property for the modules that use esnext.

I'm probably going to workaround this by including `IntlMessageFormat` on `window` through the bower package, but I haven't quite figured this out with `intl-format-cache`.

I'm curious if this is a known issue or if there is some way to bundle this package with browserify and have it work in IE8.

### PR DESCRIPTION
**Pull request by ericf**
Fixes #90

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

File: index.js [hunk 1]
```diff
@@ -12,4 +12,4 @@ require('./lib/locales');
 // locale data registered, and with English set as the default locale. Define
 // the `default` prop for use with other compiled ES6 Modules.
 exports = module.exports = IntlMessageFormat;
-Object.defineProperty(exports, 'default', {value: exports});
+exports['default'] = exports;
```
Comment (ericf) [inline index.js]: I could conditionally use O.dP here if we prefer the `default` prop to be non-enum, etc. in ES5+ environments.

Comment (caridy) [inline index.js]: na, we should be ok with enum.

**Comment by caridy** [comment]
LGTM

File: index.js (hunk 1 above)
Comment (ericf) [inline index.js]: We can also revisit this after looking into and understanding how 6to5/esnext deals (or will deal with) inter-module-system deps.

**Pull request merged**